util/packaging/aur/chapel/PKGBUILD

```diff
@@ -0,0 +1,34 @@
+
+#    Maintainer: J. Emiliano Deustua <edeustua@gmail.com>
+# Co-maintainer: Gabriel Brown <gabriel.h.brown@gmail.com>
+
+pkgname=chapel
+pkgver=1.26.0
+pkgrel=1
+pkgdesc="Programming language designed for productive parallel computing at scale"
+url="https://chapel-lang.org/"
+arch=('x86_64')
+license=('Apache')
+depends=('python' 'perl' 'llvm' 'clang')
+makedepends=('git' 'cmake')
+source=("https://github.com/chapel-lang/chapel/releases/download/${pkgver}/chapel-${pkgver}.tar.gz")
+sha256sums=('ba396b581f0a17f8da3f365a3f8b079b8d2e229a393fbd1756966b0019931ece')
+
+build() {
+        cd "$srcdir/${pkgname}-${pkgver}"
+        export CHPL_LIB_PIC=pic # remove on next release a la
+        # https://github.com/chapel-lang/chapel/pull/19785
+        ./configure --prefix=/usr
+        make
+}
+
+check() {
+        cd "$srcdir/${pkgname}-${pkgver}"
+        export PATH="$srcdir/${pkgname}-${pkgver}/bin/linux64-x86_64:$PATH"
+        make check
```

Comment:
Yes, that's my understanding as well, and I think that could be the case.

It is possible for me to include a post install script, but I'm not sure how integrated it is in the build and install process, since the package will already be installed. The one example I have hear is people using this script to bring up end user license agreement and so on, but I'm sure we can get it do more. What exactly were you thinking would go in here?

Reply:
I was just thinking it would be nice to have some scripting so that the maintainer of the Chapel AUR package can easily check that the installation is working. I.e. it could just be a script that you know how to run after installing. If we have that, we can (eventually) get some nightly testing config to check it as well.

I don't know whether or not it would be reasonable to do this with an AUR post install script but I suspect you wouldn't want people just installing the package to always run it. (So then not a post-install).

Reply:
I agree post-install is probably not the right tool here, but something like you describe would be nice.

I think the typical approach for AUR packages is to do verification with `make check` before install, but that didn't work for us. I'm sure there are issues with this approach, but the way I was going to verify that the installation is working is just by building and running it myself (then addressing user issues as they appear in comments on AUR). This is less of a problem for `chapel` since it's frozen until the next release, but perhaps would be problematic for `chapel-git` which changes (and possibly breaks) with every PR.